.vsts-ci/misc-analysis.yml

```diff
@@ -50,17 +51,14 @@ phases:
 
   - bash: |
       mdspell '**/*.md' '!**/Pester/**/*.md' --ignore-numbers --ignore-acronyms --report --en-us;
-    displayName: Test Spelling
+    displayName: Test Spelling in Markdown Files
     condition: succeededOrFailed()
 
   - powershell: |
       Import-module ./build.psm1
       $path = Join-Path -Path $pwd -ChildPath './commonTestResults.xml'
       $results = invoke-pester -Script ./test/common -OutputFile $path -OutputFormat NUnitXml -PassThru
       Write-Host "##vso[results.publish type=NUnit;mergeResults=true;runTitle=Common Tests;publishRunAttachments=true;resultFiles=$path;]"
-      if($results.TotalCount -eq 0 -or $results.FailedCount -gt 0)
-      {
-        throw "markdown link tests failed"
-      }
+      if($results.TotalCount -eq 0 -or $results.FailedCount -gt 0) { throw "Common tests failed" }
```

Comment:
please change this back.
The standard is to have the code block for conditionals on a separate line.

Reply:
Fixed in 20391a3